pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -277,6 +250,17 @@ func (rc *reconciler) attachDesiredVolumes() {
 				}
 				continue
 			}
+
+		} else {
+
+			// Don't even try to start an operation if there is already one running for the given volume and node.
+			if rc.attacherDetacher.IsOperationPending(volumeToAttach.VolumeName, "" /* podName */, volumeToAttach.NodeName) {
+				if klog.V(10) {
+					klog.Infof("Operation for multi-attach volume %q is already running for node %q. Can't start attach", volumeToAttach.VolumeName, volumeToAttach.NodeName)
```

Comment:
i think operation here is not necessarily must be a multi-attach op. maybe update the comment to avoid confusion.

Reply:
ACK, removing "multi-attach"